app/views/marketing/index.html.erb

```diff
@@ -4,10 +4,12 @@
   <div class="content-wrapper-narrow">
     <h2><%= t(".heading") %></h2>
     <p class="hero__subheading"><%= t(".subheading") %></p>
-    <form action="">
-      <input type="text">
-      <input type="submit">
-    </form>
+
+    <%= simple_form_for @registration, url: new_registration_path, method: :get do |f| %>
+      <%= f.input :zipcode %>
+
+      <%= f.submit t(".donate") %>
```

Comment:
This might be a smell that we need to extract a separate model.

In this app, there's a wizard-like registration process.

This form represents the pre-registration Zipcode submission, which acts as a guard for the rest of the registration process.

If we were to extract a separate (non-AR) model to represent this, what would you call it?